etna/ciim/client.py

```diff
@@ -107,8 +110,24 @@ def prepare_filter_aggregations(items: Optional[list]) -> Optional[str]:
             )
         else:
             filter_prepared = field + ":" + value
+
         filter_prepared_list.append(filter_prepared)
 
+    # if number of occurrences of field_for_or is more than 1, update add or to those values
+    # ["collection:<value1>:or", "collection:<value2>:or", "group:<value3>"]
+    for field in fields_using_or_operator:
+        # more than 1 value for the field
+        if sum((item.split(":", 1)[0].count(field) for item in items)) > 1:
+            # append or to the value for each field
+            updated_list_for_or_operator = []
+            for item in filter_prepared_list:
+                if item.split(":", 1)[0] == field:
+                    updated_list_for_or_operator.append(item + ":or")
+                else:
+                    updated_list_for_or_operator.append(item)
+
+            filter_prepared_list = updated_list_for_or_operator
```

Comment:
Is it not possible to do this in the above loop? And can we not just always add `:or` whether multiple values have been supplied or not, or does that cause an error?

Reply:
Don't want to "mash" adding `:or` whilst preparing the filter to remove spaces in the above loop. But as a role of preparing the filter, added after remove spaces, to keep it simple and target only those configured fields in var  `fields_using_or_operator`